### PR TITLE
temp fix issue with numpy

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,6 +48,7 @@
         "amazon-braket-schemas ~= 1.10",
         "amazon-braket-default-simulator ~= 1.9",
         "boto3-stubs",
+        "numpy == 1.23.3",
```
Comment (cqc-alec) [inline setup.py:51]: If this is only a problem with Windows and Python 3.10 then I'd probably prefer to temporarily disable tests on that combination. Pinning the numpy version is rather intrusive for people who have other packages installed in the same environment. Either way we should keep the issue open until we have a proper fix.

Comment (cqc-melf) [reply setup.py:51]: Yes I agree on this, I first wanted to check if this solves the problem on the CI, too. Will update this PR.

**Comment by cqc-melf** [comment]
@cqc-alec What do you think about this solution? It is not changing the requirements and still running the workflow.

**Review by cqc-alec — APPROVED** [review]
This will do for now, we should check again when a new amazon-braket-sdk release appears to see if we can revert it.

**Pull request merged**